packages/build/src/js-transform.ts

```diff
@@ -29,8 +29,30 @@ import isWindows = require('is-windows');
 // minifying but not compiling?
 
 // Syntax and transform plugins for ES2015.
-const babelPresetEs2015NoModules =
-    [require('@babel/preset-es2015'), {modules: false}];
+const customBabelPresetEs2015 = {
+  plugins: [
+    [require("@babel/plugin-transform-template-literals"), { loose: false, spec: false }],
```

Comment:
I think all of these parameters are default false except for the regenerator ones? Could just leave off the ones that aren't changing a default.

Reply:
You were right, almost all of these were the defaults.